inlong-sort/sort-formats/format-json/pom.xml

```diff
@@ -78,6 +78,7 @@
         <dependency>
             <groupId>io.debezium</groupId>
             <artifactId>debezium-core</artifactId>
+            <scope>provided</scope>
```

Comment:
suggested remove it if not format-json necessary if it is just for mysql cdc

Reply:
it maybe useful for future sink